docs/content/docs/glossary.md

```diff
@@ -9,7 +9,7 @@ aliases:
 
 The `@fluidframework/aqueduct` package is a library for building Fluid objects and Fluid containers within the Fluid
 Framework. Its goal is to provide a thin base layer over the existing Fluid Framework interfaces that allows developers
-to get started quickly. [Learn more.](https://fluidframework.com/docs/apis/aqueduct/)
```

Comment:
I think we need a task to clean up the `Glossary` in general. Aqueduct isn't on the API so we don't need to talk about it. Same for Code loader.

Let's tackle it in a separate PR though.